Update of counters.md documentation (#496)
MewsSystems/gitbook-connector-api: changelog/README.md

# Changelog

## 12th September 2023

> * Removed 'AccountingCounter' as a discriminator from [Counter Type Discriminator](../operations/counters.md#counter-type-discriminator) since it would never get returned nor it exists in our database.

Reviewer: ```suggestion
* Removed `AccountingCounter` from [Counter Type Discriminator](../operations/counters.md#counter-type-discriminator) since it is never used.
```
